### PR TITLE
Various shuttle fixes, new shuttle type

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Deletes useless tat in map files that can't even be loaded by anyone
Fixes various shuttle issues such as incorrect object directions, bad diagonal smooths, outdated items we dont use anymore
Adds jungle shuttle
Adds pre-lit bonfire subtype for jungle shuttle

## Why It's Good For The Game
These shuttles, while used rarely by admins, look like ultra-garbage when they are used.
Tidies up useless tat in mapping files.
New shuttle is cool and has monkeys, snakes and bunnies.

## Images of changes
(the naked humans are monkeys)

## Changelog
:cl:
add: Added a new admin emergency shuttle
fix: Fixed multiple issues with admin-selectable shuttles
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/towercap.dm [hunk 1]
```diff
@@ -119,6 +119,15 @@
 /obj/structure/bonfire/dense
 	density = TRUE
 
+/obj/structure/bonfire/lit //haha empty define
```
Comment (Sirryan2002) [inline code/modules/hydroponics/grown/towercap.dm:122]: technically this is redundant because you're already defining a child type of this

**Comment by S34NW** [comment]
> Could be a good place for the bamboo benches and such if they are buckleable (or other bamboo stuff! It's underused!!)

Great idea

File: code/modules/hydroponics/grown/towercap.dm (hunk 1 above)
Comment (S34NW) [reply code/modules/hydroponics/grown/towercap.dm:122]: Is it better to have a shell type explicitly defined, or to imply it exists?

Comment (Sirryan2002) [reply code/modules/hydroponics/grown/towercap.dm:122]: that's a good question honestly, I'm curious what other people think cause I don't know our standards for it (or if we have them). For code clarity sake, it's probably better to explicitly define it?

Comment (SteelSlayer) [reply code/modules/hydroponics/grown/towercap.dm:122]: We usually prefer to explicitly define it.

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**